app/views/topics/show.html.erb

```diff
@@ -13,6 +13,11 @@
     <% else %>
 		  <span class="bookmark"><%= b.url %></span>
     <% end %>
+    <% if current_user %>
```

Comment:
Prefer `if user_signed_in?` over `if current_user` as it explains more clearly what the intention of the code is. In this case the intention is we only want to show the like partial if a `user_` is `signed_in`.